.github/workflows/CI.yml

```diff
@@ -39,4 +39,5 @@ jobs:
       if: matrix.os != 'windows-latest'
       run: |
           # Task to auto update version to the next development iteration
-          ./gradlew updateVersion -DnewVersion=2.15.0-SNAPSHOT
+          ./gradlew updateVersion -DnewVersion=2.15.0-SNAPSHOT && ./gradlew build
```

Comment:
@rursprung fyi, I think it becomes more useful now

Reply:
yeah, this probably makes more sense!

also, if you wan to do this here you might want to add it as a commented-out version on `main` as well so that you can switch to that once `3.0.0` has been released? otherwise it'll be forgotten there

Reply:
For `main` we don't have "next" version yet ...

Reply:
not yet, but once 3.0 has been released you'll have it. and i think if you don't add it (commented-out) there now as well chances are, that it'll be forgotten until then (alternatively you could raise an issue and hope that it's being spotted once 3.0 is being released?)

Reply:
I won't be too worried about that to be fair :-) it is going to come up naturally once the time comes